### PR TITLE
create login and signup page and css

### PR DESCRIPTION
**Pull request by Ayush-mishra123**

### CODE REVIEW

File: blogfrontend/src/App.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import "./App.css"
+
+import Signup from './components/Signup';
+import Login from './components/Login';
+import {
+  BrowserRouter,
+  Routes,
+  Route,
```
Comment (rajat-thoughtwin) [inline blogfrontend/src/App.js:8]: All library imports should be on top and all local imports should be in the bottom with a line space between library imports and local imports.

File: blogfrontend/src/App.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+} from "react-router-dom";
+function App() {
+  return (
+    <>
+
```
Comment (navinwadbude) [inline blogfrontend/src/App.js:13]: why there are so much of blank spaces. Please remove them.

File: blogfrontend/src/App.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import {
+  BrowserRouter,
+  Routes,
+  Route,
+} from "react-router-dom";
+function App() {
+  return (
+    <>
+   
+
```
Comment (rajat-thoughtwin) [inline blogfrontend/src/App.js:14]: Please remove unnecessary new lines.

File: blogfrontend/src/components/Login.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
+import React,{useState} from 'react'
+import { Link } from "react-router-dom";
+
+const Login = () => {
+    const [username,setUserName]=useState({
```
Comment (rajat-thoughtwin) [inline blogfrontend/src/components/Login.js:5]: the naming convention is not proper as you are assigning object and the variable name is username it should be userDetails.

File: blogfrontend/src/components/Login.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+        console.log(name,value)
+        setUserName({
+            ...username,
+            [name]:value
+        })
+        
+    }
+    const handleSubmit=(e)=>{
+        e.preventDefault()
+        console.log("datZ",e.target.name);
```
Comment (rajat-thoughtwin) [inline blogfrontend/src/components/Login.js:20]: Please remove console logs

File: blogfrontend/src/components/Login.js [hunk 6]
```diff
@@ -0,0 +1,59 @@
+import React,{useState} from 'react'
```
Comment (navinwadbude) [inline blogfrontend/src/components/Login.js:1]: please create separate file for validation

Comment (navinwadbude) [reply blogfrontend/src/components/Login.js:1]: why are you using alert for wrong information

File: blogfrontend/src/components/Signup.js [hunk 7]
```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react";
```
Comment (navinwadbude) [inline blogfrontend/src/components/Signup.js:1]: remove unnecessary comment code. and use proper indentation

File: blogfrontend/src/index.js [hunk 8]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+import { BrowserRouter } from "react-router-dom";
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <BrowserRouter>
+
```
Comment (navinwadbude) [inline blogfrontend/src/index.js:10]: remove blank spaces

File: blogfrontend/src/components/Login.js [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+            [name]:value
+        })
+        
+    }
+    const handleSubmit=(e)=>{
+        e.preventDefault()
+        console.log("datZ",e.target.name);
+        if(!username.username){
+          alert('Enter your  username')
+        }else if(!username.password){
```
Comment (rajat-thoughtwin) [inline blogfrontend/src/components/Login.js:23]: if you are using if and else if please follow the indentation mentioned in below image.
Screenshot 2022-07-07 at 12 26 21 PM

File: blogfrontend/src/components/Login.js [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+            alert('Emter your password')
```
Comment (rajat-thoughtwin) [inline blogfrontend/src/components/Login.js:24]: Please don't use alert rather you can use toast or error below the field.

**Review by navinwadbude — CHANGES_REQUESTED** [review]

File: blogfrontend/src/components/Login.js [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+          }
+    }
+  return (
+    <div>
+    <div className="login">
+    <span className="loginTitle">Login page</span>
+    <form className="loginForm"   onSubmit={handleSubmit}  >
+      <label>Username</label>
+      <input
```
Comment (rajat-thoughtwin) [inline blogfrontend/src/components/Login.js:33]: Input must be a component which will be reused in all components.

File: blogfrontend/src/components/Signup.js [hunk 12]
```diff
@@ -0,0 +1,93 @@
...
+import { Link } from "react-router-dom";
+
+const Signup = () => {
+  const [username, setUserName] = useState({
```
Comment (rajat-thoughtwin) [inline blogfrontend/src/components/Signup.js:30]: please follow the proper naming convention as mentioned beflore.

File: blogfrontend/src/components/Signup.js [hunk 13]
```diff
@@ -0,0 +1,93 @@
...
+      ...username,
+      [name]: value,
+    });
+  };
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log("datZ", e.target.name);
+    if (!username.username) {
+      alert("Enter your  username");
+    } else if (!username.email) {
```
Comment (rajat-thoughtwin) [inline blogfrontend/src/components/Signup.js]: you have not validated the email. Like you need to check it should contain @ . etc.

File: blogfrontend/src/components/Login.js [hunk 14]
```diff
@@ -0,0 +1,59 @@
...
+        type="text"
+        className="loginInput"
+        placeholder="Enter your username..." name='username'
+        value={username.username} onChange={handlChanges}
```
Comment (navinwadbude) [inline blogfrontend/src/components/Login.js:37]: use an arrow function.

**Review by rajat-thoughtwin — CHANGES_REQUESTED** [review]
@Ayush-mishra123 please follow the proper folder structure.

**Pull request closed without merging**